frontend/pages/shared-components/enrollment/header/header.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react' 
+import styles from './header.module.css'
+import { InfoTooltip, Logo } from '../../index'
+import { TooltipTypes } from '../../../../models'
```

Comment:
two things. 
1. having many directories creates a higher stack height in the directory tree. Can we get rid of the /enrollment directory and just have the components in the shared-components/ directory? 

2. Is this just the skeleton for the header component? Someone else took on the story for the header component so hopefully this wont step on too many toes.

Reply:
disregard 2.

Reply:
Oh okay! Sounds good - I'll squash the enrollment subdirectory in my next push!